state/state_test.go

```diff
@@ -4664,3 +4664,12 @@ func (s *StateSuite) testOpenParams() state.OpenParams {
 		MongoSession:       s.Session,
 	}
 }
+
+func (s *StateSuite) TestControllerTimestamp(c *gc.C) {
+	clock := gitjujutesting.NewClock(testing.NonZeroTime())
+	err := s.State.SetClockForTesting(clock)
+	c.Assert(err, jc.ErrorIsNil)
+	got, err := s.State.ControllerTimestamp()
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(got, gc.NotNil)
```

Comment:
Here we are generally testing to see if we get something back from `s.State.ControllerTimestamp` (doesn't really matter what)? Should this test do more? If not does the test really need to exist?